taskvine/src/bindings/python3/ndcctools/taskvine/task.py

```diff
@@ -1011,5 +1011,14 @@ def __init__(self, fn, name):
         Task.__init__(self, fn)
         self.provides_library(name)
 
+        # Set some default resources to a library task so it doesn't consume whole worker.
```

Comment:
Let's not set default resources in `task.py`, I think it's going to be very easy to forget that they are here.
You are already setting them in the application code anyway.